candies/cli/parsers/standard.py

```diff
@@ -105,25 +106,13 @@ def configured(new: Callable, command: Command, prefix: str = '.') \
                  description=command.description,
                  formatter_class=help(command))
 
-    def is_(x: inspect.Parameter, of: type) -> bool:
-        return isinstance(x, of) or x == of
+    def short_of(x: Union[Type, Any]) -> Optional[str]:
```

Comment:
I believe you could simply use `x: Any`.

Reply:
Agreed. Fixed!